src/explore-education-statistics-admin/src/routes/methodologyRoutes.ts

```diff
@@ -11,28 +11,29 @@ export interface MethodologyRouteProps extends RouteProps {
 
 export type MethodologyRouteParams = {
   methodologyId: string;
+  publicationId: string;
 };
 
 export const methodologySummaryRoute: MethodologyRouteProps = {
-  path: '/methodologies/:methodologyId/summary',
+  path: '/publication/:publicationId/methodology/:methodologyId/summary',
```

Comment:
This does tie us into always needing a Publication for a Methodology.  Maybe one to just check with Cam, to see if there's a need for "global" methodologies at all?